project/MimaExcludes.scala

```diff
@@ -60,6 +60,10 @@ object MimaExcludes {
               "org.apache.spark.ml.regression.LeastSquaresCostFun.this"),
             ProblemFilters.exclude[MissingMethodProblem](
               "org.apache.spark.ml.classification.LogisticCostFun.this"),
+            ProblemFilters.exclude[MissingMethodProblem](
```

Comment:
The type of arguments in the constructor of `LogisticCostFun` are changed, so MiMa will report that the original constructor is missing.